src/Facebook/InstantArticles/Client/Helper.php

```diff
@@ -8,26 +8,24 @@
  */
 namespace Facebook\InstantArticles\Client;
 
-use Facebook\Exceptions\FacebookResponseException;
-use Facebook\Facebook;
-use Facebook\Exceptions\FacebookSDKException;
-use Facebook\Authentication\AccessToken;
+use \Facebook\Exceptions\FacebookResponseException;
+use \Facebook\Facebook;
+use \Facebook\Exceptions\FacebookSDKException;
+use \Facebook\Authentication\AccessToken;
```

Comment:
The conflict we have here is just because our namespace also starts with Facebook.
But then, to the point of reconfiguring the autogen/class loader, I couldn't get to a place where I'd have it working for both scenarios.

The /client was dropped from internal version (not needed) then, this was the only place I used for used code.

Also only classes from the standard API from hhvm/php is using, just like: `\Map`, `\DOMNode`, `\DOMDocument` etc...